mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
```

Comment:
Synthesizing a `defaultdict` and then throwing it away is a slightly odd way to implement this.

A more straightforward way to do this would be to move `test_check_responses` out to a global variable, declare it at the top level of the module, as `test_check_responses = { "agent.cpu": [ ... ]}` and then have this function simply do `from copy import deepcopy; deepcopy(test_check_responses.get(check_type, ""))`.